common/service/config/config.go

```diff
@@ -225,8 +225,12 @@ type (
 	CustomDatastoreConfig struct {
 		// Name of the custom datastore
 		Name string `yaml:"name"`
-		// ConnectAttributes is a set of key-value attributes that can be used by AbstractDatastoreFactory implementation
-		ConnectAttributes map[string]string `yaml:"connectAttributes"`
+		// ApplicationID describes the name of the application using this datastore. (e.g. cadence)
+		ApplicationID string `yaml:"applicationId"`
+		// Instance is the name of the datastore instance being used.
+		Instance string `yaml:"instance"`
```

Comment:
What's the difference between `Name` and `Instance`?

Reply:
Name is the name of the configuration, while instance is an actual instance that connection will be made to.

In case of docstore we'll have something like:
```
  datastores:
    docstore-default:
      customDatastore:
        name: "docstore"
        applicationId: "cadence"
        instance: "minimez"
```
in the test config.